### PR TITLE
feat: query data of a single pull request

### LINKED ISSUE #327: Bind querying a PR/MR data
We need to know if a deployable PR/MR comes from a fork or not. We need to add a way close to the [manual deploy](https://github.com/Scalingo/go-scalingo/blob/2e56150d070871cb404eaec09cf0baf423eafd05/scm_repo_link.go#L179) where it will query the [deployable merge requests](https://github.com/Scalingo/api/blob/2afe96050c955318c03f4f5f6cf6fa005f653dae/app/controllers/app/scm_repo_links_controller.rb#L246).

Required before beggining : https://github.com/Scalingo/api/issues/2173

### PR DESCRIPTION
**Pull request by yohann-bacha**
Fixes #327 

- [x] Add a [changelog entry](https://changelog.scalingo.com/)

### CODE REVIEW

File: scm_repo_link.go [hunk 1]
```diff
@@ -97,6 +99,20 @@ type SCMRepoLinkReviewAppsResponse struct {
 	ReviewApps []*ReviewApp `json:"review_apps"`
 }
 
+type SCMRepoLinkPullRequestResponse struct {
+	Pull RepoLinkPullRequest `json:"pull"`
```
Comment (EtienneM) [inline scm_repo_link.go:103]: question: I know it's not related to this pull request, but why does the API only returns `pull` and not `pull_request`?

File: scm_repo_link.go [hunk 2]
```diff
@@ -165,6 +181,19 @@ func (c *Client) SCMRepoLinkDelete(ctx context.Context, app string) error {
 	return nil
 }
 
+func (c *Client) SCMRepoLinkPullRequest(ctx context.Context, app string, number int) (*RepoLinkPullRequest, error) {
+	var res SCMRepoLinkPullRequestResponse
+	err := c.ScalingoAPI().DoRequest(ctx, &http.APIRequest{
+		Method:   "GET",
+		Endpoint: "/apps/" + app + "/scm_repo_link/pulls/" + strconv.Itoa(number),
```
Comment (EtienneM) [inline scm_repo_link.go:188]: nitpick: I find the following more readable 
```suggestion
		Endpoint: fmt.Sprintf("/apps/%s/scm_repo_link/pulls/%d", app, number),
```

File: scm_repo_link.go (hunk 1 above)
Comment (yohann-bacha) [reply scm_repo_link.go:103]: Well, as the `/pulls` endpoint returns a `pulls` attribute, and not `pull_requests`, I went for `pull`!

File: scm_repo_link.go [hunk 3]
```diff
@@ -2,6 +2,7 @@ package scalingo
 
 import (
 	"context"
+	"strconv"
```
Comment (EtienneM) [inline scm_repo_link.go:5]: No longer needed, I hope the CI will catch this.

**Review by EtienneM — DISMISSED** [review]
Just a comment to fix, otherwise LGTM

**Review by EtienneM — APPROVED** [review]

**Pull request merged**